### PR TITLE
fix(listbox): changed default selected value to be blank

### PR DESCRIPTION
**Pull request by agliga**
## Description
Fixed listbox to not selected a default option. 
Added an `unselectedText` attribute to default the initial text when no option is selected. (Should default to - like in skin).
Fixed tests and made sure they work with the new missing attribute. Added a new mock to check for first item selected (As most tests are)

## References
https://github.com/eBay/ebayui-core/issues/1599

## Screenshots
Screen Shot 2022-01-21 at 11 53 54 AM
Screen Shot 2022-01-21 at 11 53 56 AM
Screen Shot 2022-01-21 at 11 54 00 AM

### CODE REVIEW

**Review by ArtBlue — APPROVED** [review]

File: src/components/ebay-listbox/test/mock/index.js [hunk 1]
```diff
@@ -12,6 +12,15 @@ exports.Basic_3Options = {
     })),
 };
 
+exports.Basic_3Options_FirstSelected = {
```
Comment (PrashantAshok) [inline src/components/ebay-listbox/test/mock/index.js:15]: would it be helpful to add a test case where none of the options are selected?

Comment (agliga) [reply src/components/ebay-listbox/test/mock/index.js:15]: I have one on server. It checks if no items are selected.

**Review by ianmcburnie — APPROVED** [review]
Nice! :-)
